plugins/ltac/g_class.mlg

```diff
@@ -22,22 +22,27 @@ DECLARE PLUGIN "ltac_plugin"
 
 {
 
-let set_transparency cl b =
+let set_transparency locality cl b =
+  let mod_locality = match locality with
+    |  Goptions.OptDefault -> if Locality.make_section_locality None then Goptions.OptGlobal else Goptions.OptLocal
```

Comment:
```suggestion
    |  Goptions.OptDefault -> if not (Global.sections_are_opened ()) then Goptions.OptGlobal else Goptions.OptLocal
```

Reply:
Should I just swap the order instead of having a not?